src/tz_macos.rs

```diff
@@ -4,13 +4,17 @@ pub(crate) fn get_timezone_inner() -> Result<String, crate::GetTimezoneError> {
     // Get string like ""Europe/Berlin (GMT+2) offset 7200 (Daylight)""
     let mut str1 = format!("{:?}", tz);
 
-    // strip leading double quotes
-    while str1.starts_with('"') {
-        str1 = str1[1..].to_string();
-    }
+    // We split the string first, so removing the leading quotes has to move fewer characters.
+    str1.truncate(match str1.split_ascii_whitespace().next() {
+        Some(s) => s.len(),
```

Comment:
I put the truncation outside of the match. Inside the Some() arm, the string is still borrowed. Cf. https://github.com/rust-lang/rust/issues/59159